Makefile

```diff
@@ -1,6 +1,13 @@
 GO_FILES?=$$(find . -name '*.go' |grep -v vendor)
 TAG?=latest
 
+.GIT_COMMIT=$(shell git rev-parse HEAD)
```

Comment:
Can you explain this change and why it's needed specifically for this PR?

Reply:
I got tired of looking up and recreating the install command, using docker redist to build for local testing is a huge time drag because it is so slow.  I can remove this if you want

Reply:
@alexellis if I remove the changes with the makefile, can we move this forward?

Reply:
So this doesn't affect the Dockerfile, just your local way of building?

Reply:
If it only affects the new local install command I think it's fine to leave in.

Reply:
Purely the `local-install` command.  The variables are also hidden in the `make` tab completion because they begin with a dot.